test_project_geo/sample/tests.py

```diff
@@ -0,0 +1,19 @@
+import sys
```

Comment:
unfortunately this tests are not work with our CI. And I do not see a reason to add one more test project.